statesync/reactor.go

```diff
@@ -265,6 +268,7 @@ func (r *Reactor) Sync(stateProvider StateProvider, discoveryTime time.Duration)
 		r.mtx.Unlock()
 		return sm.State{}, nil, errors.New("a state sync is already in progress")
 	}
+	r.metrics.Syncing.Add(1)
```

Comment:
Do the state sync metrics have different semantic from those of block sync? Because here you use `.Add` vs for the blocksync metric where you use `.Set`. Same below at the end of the function where you use `Add(0)`.

Reply:
Good observation. I was just copying them across. I think it makes sense to use `Set`